_gabarits-jsonld/README.md

```diff
@@ -47,9 +47,15 @@ Saisissez des types supplémentaires correspondant au type particulier de l’é
 ### [_name_](https://schema.org/name)
 Saisissez le titre de l’événement.
 
+### @id
```

Comment:
@fjjulien Generating URIs can be good, but it can also be bad if done poorly. 

In my opinion we should encourage using existing URIs rather than creating ones on-the-fly.

Structured data without URIs is better than structured data with bad URIs. The most extreme case of this is the Place des Arts website that has all their events with the same URI making the structured data almost impossible to load into a triple store. It would be much better if they didn't create any URIs.  

When loading structured data without URIs, the triple store generates blank nodes. Although blank nodes cannot be referenced externally like URIs, they do enable the data to be used, queried and traversed using SPARQL.  Food for thought.

Also, it would be good to have a sameAs for the event using an Artsdata URI. Artsdata URIs should not be used in the @id position as a subject outside of Artsdata, rather as objects in a sameAs relation.

Reply:
I can edit the instructions further to insist on the need to append both the date AND time. I can also stress even further that these IDs have to be unique.

Reply:
I will also indicate clearly that no ID is better than an ID that isn't unique. I'll resume this tomorrow morning.